app/views/layouts/angular-bootstrap/_auth_credentials_angular_bootstrap.html.haml

```diff
@@ -91,7 +79,7 @@
             = _("Required")
           %span.help-block{"ng-show" => "!angularForm.#{prefix}_verify.$error.required && #{prefix}_VerifyCtrl != undefined && #{prefix}_VerifyCtrl.$error.verifypasswd"}
             = passwd_mismatch
-    %div{"ng-show" => "showVerify('#{prefix}_userid')"}
+    %div{"ng-show" => "showValidate('#{prefix}')"}
```

Comment:
I am building on top of #8503 - and that is included as first commit in this PR - there were two identical blocks of code one showing when `showVerify` returned `true` and other showing when it returned `false` called with the same arguments so validate buttons were displayed always.

@blomquisg you are right - `showValidate` function is used to determine if validate button will be displayed at all - but b/c of double block mentioned above we do not have to include `showVerify` into our thoughts - only `showValidate` as it is defined in controller.